src/emails/templates/breachAlert/BreachAlertEmail.tsx

```diff
@@ -253,15 +244,21 @@ export const RedesignedBreachAlertEmail = (
             )
           }
         </mj-section>
-        <DataPointCount
-          {...props}
-          utmCampaignId={utmCampaignId}
-          utmContentSuffix={utmContentSuffix}
-        />
+        {props.subscriber.onerep_profile_id &&
```

Comment:
I don't _expect_ that someone's OneRep profile ID can be 0, but better safe than sorry. Who knows, you might save someone a lot of time debugging an issue when they just have a sandbox account with that ID :)

```suggestion
        {typeof props.subscriber.onerep_profile_id === "number" &&
```